js/main.js

```diff
@@ -25,5 +26,76 @@ function createEntry(event) {
   data.nextEntryId++;
   data.entries.unshift(entryObj);
   $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $entryList.prepend(renderEntry(entryObj));
   $journalEntry.reset();
 }
+
+function renderEntry(entryObj) {
+  var ulEl = document.createElement('ul');
```

Comment:
Your `renderEntry` function is creating a whole new ul for every entry, but the HTML should have the ul in it already and all entries should be a li element.

Reply:
Fixed the renderEntry function. Now it is making one li for every entry and the HTML has a ul element in it.